### PR TITLE
Bugfix/issue5

### PR DESCRIPTION
**Pull request by normann**
This is for #5 with 2 commits,
each commit is targeting fix a sub-issue under #5

### CODE REVIEW

File: src/extensions/MetaDescriptionFallbackExtension.php [hunk 1]
```diff
@@ -96,7 +97,12 @@ public function getGeneralMetaDescription($dataObject = null)
         // add a space to closing </p> to prevent bunching, strip all tags and replace multiple spaces with a single one
         $metaDescription = preg_replace('/\s+/', ' ', strip_tags(trim(str_replace('</p>', '</p> ', $metaDescription))));
 
-        return $metaDescription;
+        // call ATT() on an instance of DBText's family, preferablly the DBMarkdownText DB fieldtype which is known
+        // to address Markdown format by Chrometoaster
+        $className = class_exists('Chrometoaster\MarkdownEnabledText\FieldTypes\DBMarkdownText') ?
+            \Chrometoaster\MarkdownEnabledText\FieldTypes\DBMarkdownText::class : DBText::class;
+
+        return singleton($className)->setValue($metaDescription)->ATT();
```
Comment (michalkleiner) [inline src/extensions/MetaDescriptionFallbackExtension.php]: Please just use `DBText::class` directly. If needed, a custom class can be provided via Injector and is out of scope of this module.

**Review by michalkleiner — CHANGES_REQUESTED** [review]

**Comment by normann** [comment]
Done the requested changes @michalkleiner

**Review by michalkleiner — APPROVED** [review]

**Comment by michalkleiner** [comment]
Thanks @normann, feel free to merge and tag a new minor release.

**Comment by normann** [comment]
@michalkleiner just confirm you want me to tag a **minor** release, rather than a **patch** release?

**Comment by michalkleiner** [comment]
@normann yep, you're right, it can be just a **patch** version.

**Pull request merged**